list.md

```diff
@@ -7,175 +7,68 @@ librarians.
+| Germany     |                 | [BVB-KOBV-ALLIANZ](https://github.com/bvb-kobv-allianz)                                                               |              |
+| Germany     |                 | [Max Planck Digital Library - MPDL](https://github.com/MPDL)                                                          |              |
+| Germany     |                 | [FID Jüdische Studien](https://github.com/FID-Judaica)                                                                |              |
+| Germany     |                 | [Fachinformationsdienst Darstellende Kunst](https://github.com/FID-Darstellende-Kunst)                                |              |
+| Luxembourg  |                 | [National Library of Luxembourg](https://github.com/natliblux)                                                        |              |
+| Netherlands |                 | [National Library of the Netherlands / Research](https://github.com/KBNLresearch)                                     |              |
+| Norway      | Bergen          | [University of Bergen Library (Norway)](https://github.com/ubbdst)                                                    |              |
+| Norway      | Oslo            | [Digital Development Team at Oslo Public Library](https://github.com/digibib)                                         |              |
+| Norway      | Oslo            | [University of Oslo Library (Norway)](https://github.com/scriptotek)                                                  |              |
+| Norway      |                 | [NTNU University Library (Norway)](https://github.com/NTNUUB)                                                         |              |
+| Norway      |                 | [BI Norwegian Business School Library (Norway)](https://github.com/bisentralen)                                       |              |
+| Switzerland |                 | [swissbib (Swiss Library Catalogue)](https://github.com/swissbib)                                                     |              |
+| Switzerland |                 | [project linked-swissbib](https://github.com/linked-swissbib)                                                         |              |
+| UK          | London          | [The British Library](https://github.com/britishlibrary)                                                              |              |
+| UK          | London                | [British Library Labs Project](https://github.com/BL-Labs)                                                            |              |
+| Sweden      |                 | [National Library of Sweden 1](https://github.com/Kungbib)                                                            |              |
+| Sweden      |                 | [National Library of Sweden 2](https://github.com/Libris)                                                            |              |
```

Comment:
Dito. And isn't Libris more than the national library, some sort of union catalogue?